src/components/main.tsx

```diff
@@ -47,17 +77,18 @@ const Main = () => {
     return (
         <Form>
             <div className="main-form">
-                <InputGroup>
+                <InputGroup style={{paddingBottom: invalidError  ? 0 : 20}}>
                     <Input
                         onChange={handleChange}
                         id="ethereumAddress"
                         spellCheck={false}
                         autoComplete="off"
                         className="main-input"
-                        placeholder="Enter your Ethereum address"
+                        placeholder="Enter your Ethereum address or ENS Domain"
```

Comment:
I just noticed this longer text doesn't fit on mobile device. You might have to target it with a @media query and shrink the font for mobile.